tests/acceptance/features/apiSharingNgLinkShare/linkShare.feature

```diff
@@ -3817,4 +3817,129 @@ Feature: Create a share link for a resource
           }
         }
       }
-      """
\ No newline at end of file
+      """
+
+
+  Scenario Outline: create quick link share of a project space drive using root endpoint
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    When user "Alice" creates the following space link share using root endpoint of the Graph API:
+      | space           | projectSpace       |
+      | permissionsRole | <permissions-role> |
+      | password        | %public%           |
+      | displayName     | Link               |
+      | quickLink       | true               |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "hasPassword",
+          "id",
+          "link"
+        ],
+        "properties": {
+          "hasPassword": {
+            "const": true
+          },
+          "id": {
+            "pattern": "^[a-zA-Z]{15}$"
+          },
+          "link": {
+            "type": "object",
+            "required": [
+              "@libre.graph.displayName",
+              "@libre.graph.quickLink",
+              "preventsDownload",
+              "type",
+              "webUrl"
+            ],
+            "properties": {
+              "@libre.graph.displayName": {
+                "const": "Link"
+              },
+              "@libre.graph.quickLink": {
+                "const": true
+              },
+              "preventsDownload": {
+                "const": false
+              },
+              "type": {
+                "const": "<permissions-role>"
+              },
+              "webUrl": {
+                "type": "string",
+                "pattern": "^%base_url%/s/[a-zA-Z]{15}$"
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role |
+      | view             |
+      | upload           |
+      | edit             |
+      | createOnly       |
```

Comment:
can we add secure view here ?

Reply:
I think there's no secure view option while creating quick link share. @amrita-shrestha

Reply:
Currently, no scenerio should be added for Secure View for sharing project drive.

Reply:
@prashant-gurung899 better if you add such information in the description. Important details should be mentioned in the description

Reply:
Sure!